### PR TITLE
Wishlist functionality

### PR DESCRIPTION
**Pull request by Mohitkumar33**
added wishlist functionality

### CODE REVIEW

File: src/contexts/wishlist-context.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { createContext, useContext, useReducer } from "react";
+
+const wishlistContext = createContext(null);
+const wishlistReducer = (wishlistState, wishlistAction) => {
+  const { wishlistItems } = wishlistState;
+  switch (wishlistAction.type) {
+    case "ADD_TO_WISHLIST":
+      // return{...wishlistState,wishlistItems:wishlistItems.concat(wishlistAction.payload)}
```
Comment (deepak29-git) [inline src/contexts/wishlist-context.js:8]: remove commented code

File: src/contexts/wishlist-context.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { createContext, useContext, useReducer } from "react";
+
+const wishlistContext = createContext(null);
+const wishlistReducer = (wishlistState, wishlistAction) => {
```
Comment (deepak29-git) [inline src/contexts/wishlist-context.js:4]: you can extract this function in another file so it would see more cleaner

**Review by deepak29-git — APPROVED** [review]
code is cleaner and also folder structure is good

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean and Well Structured

File: src/contexts/wishlist-context.js (hunk 2 above)
Comment (Mohitkumar33) [reply src/contexts/wishlist-context.js:4]: ok

**Pull request merged**